mangaki/mangaki/utils/nmf.py

```diff
@@ -0,0 +1,54 @@
+from mangaki.utils.chrono import Chrono
+from sklearn.decomposition import NMF
+from scipy.sparse import lil_matrix
+import numpy as np
+import pandas
+
+
+class MangakiNMF(object):
+    M = None
+    W = None
+    H = None
+    def __init__(self, NB_COMPONENTS=10):
+        self.NB_COMPONENTS = NB_COMPONENTS
+        self.chrono = Chrono(True)
+        self.works = pandas.read_csv('data/works.csv', header=None).as_matrix()[:, 1]
```

Comment:
By the way, we should move the `read_csv` and give them as a parameter, as all algorithms seems to use these files.